crates/sui-network/src/state_sync/mod.rs

```diff
@@ -698,11 +722,11 @@ where
             > highest_synced_checkpoint.sequence_number()
             // skip if we aren't connected to any peers that can help
             && self
-                .peer_heights
-                .read()
-                .unwrap()
-                .highest_known_checkpoint_sequence_number()
-                > Some(*highest_synced_checkpoint.sequence_number())
+            .peer_heights
+            .read()
+            .unwrap()
+            .highest_known_checkpoint_sequence_number()
+            > Some(*highest_synced_checkpoint.sequence_number())
```

Comment:
revert this?

Reply:
ok i will, it seems like a doing of cargo fmt